File: Dsa-day106.cpp
class Solution {
public:
    vector<int> getSneakyNumbers(vector<int>& nums) {
        unordered_map<int,int> freq;
        vector<int> res;
        for(int x: nums){
            freq[x]++;
            if(freq[x]==2) res.push_back(x);
        }
        return res;
    }
};

File: Dsa-day106.py
class Solution:
    def getSneakyNumbers(self, nums: List[int]) -> List[int]:
        freq = {}
        res = []
        for x in nums:
            freq[x] = freq.get(x, 0) + 1
            if freq[x] == 2:
                res.append(x)
        return res
